pkg/runtime/serializer/codec_factory.go

```diff
@@ -220,9 +220,10 @@ type DirectCodecFactory struct {
 	CodecFactory
 }
 
-// EncoderForVersion returns an encoder that does not do conversion. gv is ignored.
-func (f DirectCodecFactory) EncoderForVersion(serializer runtime.Encoder, _ runtime.GroupVersioner) runtime.Encoder {
+// EncoderForVersion returns an encoder that does not do conversion.
```

Comment:
"that does not do conversion" followed by using an argument that "// GroupVersioner refines a set of possible conversion targets into a single option." is a little weird.  Just reading straight down, so I haven't investigated usage.